.github/workflows/ci.yml

```diff
@@ -20,7 +31,17 @@ jobs:
     steps:
       - uses: actions/checkout@v4
         with:
+          repository: Eppo-exp/eppo-multiplatform
+          ref: ${{ inputs.sdk_branch || github.ref }}
           submodules: true
+
+      - uses: actions/checkout@v4
+        if: ${{ inputs.test_data_branch }}
+        with:
+          repository: Eppo-exp/sdk-test-data
+          ref: ${{ inputs.test_data_branch }}
+          path: sdk-test-data
```

Comment:
I hope that this should be enough but maybe we need to override `submodules` to `false` if `test_data_branch` is supplied